docs/index.md

```diff
@@ -84,18 +84,14 @@ prefixes in the list.
 The permitted registry prefixes are:
 
 ```
-quay.io/buildah
 quay.io/redhat-appstudio
-registry.access.redhat.com/ubi8
-registry.access.redhat.com/ubi8-minimal
-registry.redhat.io/ocp-tools-4-tech-preview
-registry.redhat.io/openshift4
-registry.redhat.io/openshift-pipelines
+registry.access.redhat.com
+registry.redhat.io
```

Comment:
Maybe a trailing slash is useful so we don't pass on registry.redhat.io-hacks.com

Reply:
Great catch. Done.